packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -100,7 +101,14 @@ class VmServiceWrapper implements VmService {
 
   @override
   Future<Success> clearCpuSamples(String isolateId) async {
-    if (await isProtocolVersionLessThan(major: 3, minor: 27)) {
+    if (await isProtocolVersionSupported(
+        supportedVersion: SemanticVersion(major: 3, minor: 27))) {
+      return _trackFuture(
+          'clearCpuSamples',
+          _vmService.clearCpuSamples(
+            isolateId,
```

Comment:
nit: remove this trailing comma? I believe this call would all fit on one line

Reply:
done.